api-spec/protobuf/ark/v1/admin.proto

```diff
@@ -63,4 +69,13 @@ message GetRoundsRequest {
 
 message GetRoundsResponse {
   repeated string rounds = 1;
-}
\ No newline at end of file
+}
+
+message UpdateMarketHourRequest {
+  int64 first_market_hour = 1;
+  int64 period = 2;
+  int64 round_lifetime = 3;
```

Comment:
We may want to specify market hours longer or shorter than an hour so let's specify the start and end time of the first market hours, the period and the round interval (not filefitme) during the market hours

```suggestion
  // Starting timestamp of the first market hour.
  int64 start_time = 1;
  // Ending timestamp of the first market hour.
  int64 end_time = 2;
  // Period expressed in hours between market hours 
  int64 period = 3;
  // Round interval during the market hours, this overrides the default one.
  int64 round_interval = 4;
```

Reply:
please reflect these changes also in env vars